### ISSUE TITLE
add Status en statics

### ISSUE THREAD

**Issue by Dominic-070** [AlexxIT/Jura#13]
First of all, thank you for your great work! I had been hoping for some time to integrate my JURA S8 into Home Assistant.

Could you add sensors with status such as machine status, product proces?

Could you add statistics?
How much coffee made? or How much until next cleaning?

Is it possible to turn on the device via Bluetooth?

**Comment by AlexxIT** [comment]
Technically, I can add that.
But I can't physically find the time to do it.

The coffee maker cannot be turned on via Bluetooth.

**Comment by proza19** [comment]
Thanks for yoy work @AlexxIT  it's working on;

Device info
E6 (EA)
by Jura
MAC: D9:3E:14:93:16:__

**Comment by bramd** [comment]
@AlexxIT Any pointers on where to start? I might have some time to implement this in the coming months and also would like to see these sensors. But first have to get the integration working properly :)

**Comment by AlexxIT** [comment]
You can use some mobile utilities for read bluetooth attributes from device

**Comment by bramd** [comment]
@AlexxIT Yes I know that, but had hoped someone had documented this part of the protocol already :).

**Comment by andrei-marinache** [comment]
I came up with this code: https://github.com/andrei-marinache/Jura
It reads alerts and statistics.
I know that the code is crap, but it works. Hopefully, it works for all the Jura machines.

Next I will try to integrate the code to this component, to make the data available in HA, but I don't know if I'm capable of doing it. Maybe @bramd or @AlexxIT can help.

**Comment by PV-Joe** [comment]
I‘d love to see the Jura integration for Home Assistant only to get the statistic data from the machine.

Hoping that you‘ll find the time to add those.
Keep up the good work.

**Comment by saschaspeidel** [comment]
+1 👍 
for me that even is the more relevant part. Hope this gets added to the integration :)

**Comment by tidalf** [comment]
Hello I've started something here https://github.com/tidalf/Jura. If someone wants to test on something other than the ENA8, it works quite well here.

**Comment by AlexxIT** [comment]
Yes. PRs are welcome. But it can take a significant amount of time to test and accept them. Many other projects

**Comment by derjoerg** [comment]
@tidalf
I just tested your release 1.3.2 with my E8 and everything is working as expected 😃 

Here are my statistics:
![Image](https://github.com/user-attachments/assets/d8327f68-7919-4673-8c87-4f8de628bf6d)

And here my alerts:
![Image](https://github.com/user-attachments/assets/25e2ca9a-0f12-4c97-acd2-eaff436889f0)

Thanks a lot

**Comment by samvandenberge** [comment]
@tidalf Works great on my E4 as well. Thanks!

![Image](https://github.com/user-attachments/assets/c882f884-1e93-4fb7-8f4f-1c8a4726dda7)

**Comment by tidalf** [comment]
AlexxIT has merged the prs 🎉 thx ! The cappu alert is fixed. I'll try to add the maintenance percents (and the "no beans" alert), it's not in the same stats call.

**Comment by AlexxIT** [comment]
https://github.com/AlexxIT/Jura/releases/tag/v1.2.0

**Comment by Dominic-070** [comment]
@andrei-marinache @tidalf Seems to work fine, Thnx.
Is there an option for the sensors such as the product counters, alerts and Diagnostic sensors to keep their current status when the coffee machine is switched off? Now when you turn off the coffee machine, the sensors go to unavailable.

**Comment by andrei-marinache** [comment]
https://github.com/andrei-marinache/Jura-AlexxIT/releases/tag/v0.1
Added maintenance counters and percents. Also added all the alarms available.
I have to tweak the code a little more, I will open an PR afterwards.

Yes, status should be kept when the machine is off, I'll look into that.
Also sometimes the alarms are cleared (on reconnect), I'll also check that.
